### ISSUE TITLE
Upload field shown as empty at the end of an upload

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#5035]
To reproduce, create a form with an upload field, publish, open the form, throttle ("3G slow" in the Chrome Dev Tools does the trick), upload a 250k file, and notice how at the end of the upload the field is left empty.

[+1 from customer](https://3.basecamp.com/3600924/buckets/19902726/messages/3914552457)

**Comment by avernet** [comment]
> Did you try the throttle with 2020.1 or master?

The customer couldn't reproduce this with 2020.1. This might be because the bug has been fixed, or because the new upload fields hide the bug. But I also checked the code in 2020.1 (as well as the latest code), and looking at the code I am not seeing there that problem identified in my previous comment, in both cases, i.e. it looks to me like we are now doing the `ajaxRequestInProgress = false` at the right time. So there is no need to also put this issue on the 2020.1.5 or 2021.1 projects.
